flink-table/flink-table-api-java/src/main/java/org/apache/flink/table/api/TableEnvironment.java

```diff
@@ -430,11 +362,51 @@ static TableEnvironment create(EnvironmentSettings settings) {
 	 * }
 	 * </pre>
 	 *
-	 * @deprecated Use {@link #sql(String)}.
+	 * <p>A DDL statement can also execute to create a table:
+	 * For example, the below DDL statement would create a CSV table named `tbl1`
+	 * into the current catalog:
+	 * <blockquote><pre>
+	 *    create table tbl1(
+	 *      a int,
+	 *      b bigint,
+	 *      c varchar
+	 *    ) with (
+	 *      connector = 'csv',
+	 *      csv.path = 'xxx'
```

Comment:
Give a validate with property? 
```
connector.type = 'filesystem',
connector.property-version = '1',
connector.path = '/path/to/file',
format.type = 'csv',
format.property-version = '1',
format.derive-schema = 'true'
```
However, I'm not sure the above properties work.

Reply:
I agree with Jark. If we show examples, then examples that also work in the current version.